src/osdag/cad/common_logic.py

```diff
@@ -24,6 +24,8 @@
 from .items.stiffener_flange import Stiffener_flange
 from .items.rect_hollow import RectHollow
 from .items.circular_hollow import CircularHollow
+from .items.double_angles import BackToBackAnglesWithGussets_same_side
+from .items.double_angles import BackToBackAnglesWithGussets_opp_side
```

Comment:
Please stick to a [consistent naming condition](https://peps.python.org/pep-0008/). For classes use `CamelCase` and `snake_case` for methods.